src/utilstrencodings.cpp

```diff
@@ -560,6 +560,14 @@ bool ParseHDKeypath(const std::string& keypath_str, std::vector<uint32_t>& keypa
         // Finds whether it is hardened
         uint32_t path = 0;
         size_t pos = item.find("'");
+        if (pos == std::string::npos) {
+            // Look for h
```

Comment:
Why?

Reply:
It's fairly standard to use `h` or `H` as the indicator, not just `'`. In fact, the BIP uses `H`, but as a subscript. `'` is not actually used as the indicator for hardened in the BIP even though that is what everyone uses.

The main reason for allowing this is so that you don't have to do any annoying escaping stuff for the `'` when entering the keypaths on the command line.